nxdrive/osi/linux/linux.py

```diff
@@ -100,3 +114,66 @@ def register_protocol_handlers(self) -> None:
             log.info(
                 f"Registered {original_executable!r} for URL scheme {NXDRIVE_SCHEME!r}"
             )
+
+    @if_frozen
+    def send_sync_status(self, state: DocPair, path: Path) -> None:
+        """
+        Send the sync status of a file to the FinderSync.
+
+        :param state: current local state of the file
+        :param path: full path of the file
+        """
+        try:
+            status = get_formatted_status(state, path)
+            if status:
+                log.debug(f"Sending status to FinderSync for {path!r}: {status}")
+                self._set_icon(status)
+        except Exception:
+            log.exception("Error while trying to send status to FinderSync")
+
+    def _set_icon(self, status: Dict[str, str]) -> None:
+        value = Status(int(status["value"]))
+        path = status["path"]
+        emblem = icon_status[value]
+        cmd = [
+            "gio",
+            "set",
+            "-t",
+            "stringv",
+            str(path),
+            "metadata::emblems",
+            emblem,
+        ]
+        try:
+            subprocess.check_call(cmd)
+        except subprocess.CalledProcessError:
+            log.warning(f"Could not set the {emblem} emblem on {path!r}")
+
+    def _icons_to_emblems(self) -> None:
+        """
+            Copy nuxeo overlay icons to linux local icons folder.
+            Previous local icons will be replaced.
+        """
+        shared_icons = Path.home() / ".local/share/icons"
+        shared_icons.mkdir(parents=True, exist_ok=True)
+
+        statuses = ["synced", "syncing", "conflicted", "error", "locked", "unsynced"]
+        icons = find_icon("") / "overlay" / "linux"
+
+        for status in statuses:
+
+            icon = icons / f"badge_{status}.svg"
+            emblem = shared_icons / f"emblem-nuxeo_{status}.svg"
+
+            try:
+                identical = filecmp.cmp(icon, emblem, shallow=False)
+            except OSError:
+                pass
```

Comment:
Maybe add a comment, something like:
```suggestion
                # Most likely the *icon* doest not exist yet
                pass
```